packages/laconia-event/src/apigateway/getContentType.js

```diff
@@ -1,5 +1,6 @@
+const getResponseProps = require("./getResponseProps");
```

Comment:
To be honest I don't know if we need this file anymore.

The only reference we have now are tests.